lib/adapter/TingClientRequestAdapter.php

```diff
@@ -69,7 +69,7 @@ public function execute(TingClientRequest $request) {
         $stopTime = explode(' ', microtime());
         $time = floatval(($stopTime[1]+$stopTime[0]) - ($startTime[1]+$startTime[0]));
 
-        $this->logger->log('Completed SOAP request ' . $soapAction . ' ' . $request->getWsdlUrl() . ' (' . round($time, 3) . 's). Request body: ' . $client->requestBodyString . ' Response: ' . $response);
+        $this->logger->log('Completed SOAP request ' . $soapAction . ' ' . $request->getWsdlUrl() . ' (' . round($time, 3) . 's). Request body: ' . $client->requestBodyString );
```

Comment:
For the record I have removed this change.

If we are to change this then please:

1. Create a separate issue and pull request for this.
2. Since the problem is Watchdog specific then trim the log message according to Watchdog limitations in the [`TingClientDrupalWatchDogLogger` class](https://github.com/ding2/ting-client/blob/master/lib/log/TingClientDrupalWatchDogLogger.php).